build.gradle

```diff
@@ -83,7 +87,7 @@ dependencies {
 bootJar {
     baseName 'rpa-dg-template-management-api'
     archiveName = "$baseName"+".jar"
-    destinationDir = file("$rootDir/build/libs")
+    destinationDir = file("$rootDir/build/lib")
```

Comment:
When running the gradle assembly job a shell script is created (I don't know where it comes from) it references the jar in `lib` not `libs` so I updated it to match